test/utils/test-swagger-gen.ts

```diff
@@ -476,15 +484,17 @@ const generateSwaggers = async (
       `--use=.`,
       `--package-name=${packageName}`,
       `--package-version=${package_version}`,
-      `--hide-clients=${!!hideClients}`
+      `--hide-clients=${!!hideClients}`,
+      `--ignore-pageable-array-check=${!!ignorePageableArrayCheck}`
     );
     if (isDebugging) {
       commandArguments.push(`--typescript.debugger`);
     }
     const generationTask = async () => {
       console.log(`=== Start ${name} ===`);
       const childProcess = spawn(autorestCommand, commandArguments, {
-        stdio: [process.stdin, process.stdout, process.stderr]
+        stdio: [process.stdin, process.stdout, process.stderr],
+        shell: process.platform === "win32"

```

Comment:
hmmm, why?

Reply:
So, the generation process works on Windows 10 machine